x-pack/plugins/task_manager/server/lib/bulk_operation_buffer.ts

```diff
@@ -21,36 +22,30 @@ export interface Entity {
   id: string;
 }
 
-export interface OperationError<Input, ErrorOutput> {
-  entity: Input;
-  error: ErrorOutput;
+export interface ErrorOutput {
+  type: string;
+  id: string;
+  error: SavedObjectError;
```

Comment:
I simplified the changes in this file so that Input === Output type and the ErrorOutput always equals this interface. Made things much simpler given a function below needed to get `id` from the `error` object now that `entity` is no longer a thing.